src/main/java/com/meilisearch/sdk/SearchRequest.java

```diff
@@ -336,4 +337,31 @@ private SearchRequest(
     public void setQuery(String q) {
         setQ(q);
     }
+
+    /**
+     * Method that returns the JSON String of the SearchRequest
+     *
+     * @return JSON String of the SearchRequest query
+     */
+    @Override
+    public String toString() {
+        JSONObject jsonObject =
+                new JSONObject()
+                        .put("q", this.q)
+                        .put("offset", this.offset)
+                        .put("limit", this.limit)
+                        .put("attributesToRetrieve", this.attributesToRetrieve)
+                        .put("cropLength", this.cropLength)
+                        .put("cropMarker", this.cropMarker)
+                        .put("highlightPreTag", this.highlightPreTag)
+                        .put("highlightPostTag", this.highlightPostTag)
+                        .put("matches", this.matches)
+                        .put("facetsDistribution", this.facetsDistribution)
+                        .put("sort", this.sort)
+                        .putOpt("attributesToCrop", this.attributesToCrop)
+                        .putOpt("attributesToHighlight", this.attributesToHighlight)
+                        .putOpt("filter", this.filter)
+                        .putOpt("filter", this.filterArray);
+        return jsonObject.toString();
```

Comment:
Adding a new empty line before the return increases the readability!
```suggestion

        return jsonObject.toString();
```